src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -142,7 +143,9 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, reimbursementAcc
     const hasMembersError = PolicyUtils.hasEmployeeListError(policy);
     const hasPolicyCategoryError = PolicyUtils.hasPolicyCategoriesError(policyCategories);
     const hasGeneralSettingsError = !isEmptyObject(policy?.errorFields?.generalSettings ?? {}) || !isEmptyObject(policy?.errorFields?.avatarURL ?? {});
-    const shouldShowProtectedItems = PolicyUtils.isPolicyAdmin(policy);
+    const currentUserPersonalDetails = useCurrentUserPersonalDetails();
+    const employee = policyProp?.employeeList?.[currentUserPersonalDetails.login ?? ''] ?? {};
+    const shouldShowProtectedItems = PolicyUtils.isPolicyAdmin(policyProp) || (employee && employee.role === CONST.POLICY.ROLE.ADMIN);
```

Comment:
I think it's better to move the fix into method `isPolicyAdmin`

Reply:
I think you can try get accountid by policy owner, get employee and then compare employee role.